Pull Request #385 Searching on multiple tags is broken

oli-ver opened the pull request:
Added simple exists sql clause to find all experiments or database
entries that contains the tag or both of the chosen tags.

I tested my suggestion and it seems to work. Perhaps we will need some option menu to choose, if the sql query should use AND or OR. At the moment it uses AND.

Please ignore the .project file... You will not need it

oli-ver commented:
#385 Should be fixed by this. Please have a look.

NicolasCARPi commented:
Hello,

Thanks for your contribution. Please remove the .project file from the commit and I'll have a look. And squash the commits.

oli-ver commented:
Hello. I removed the .project file and squashed the commits using this stackoverflow hint: http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/5201642#5201642

It this ok for you now?
